addons/common/slugify.js

```diff
@@ -0,0 +1,8 @@
+module.exports = function slugify(text) {
+  return text.toString().toLowerCase()
+    .replace(/\s+/g, '-')           // Replace spaces with -
+    .replace(/[^\w\-]+/g, '')       // Remove all non-word chars
+    .replace(/\-\-+/g, '-')         // Replace multiple - with single -
+    .replace(/^-+/, '')             // Trim - from start of text
+    .replace(/-+$/, '');            // Trim - from end of text
+}
```

Comment:
Interesting... that's a good question. 

The real motivation here is to make sure that the `<Field />`'s label properly associates itself with an input. The name itself won't be used for sending parameters. I'm not sure that we need to add the bulky logic for ensuring full compliance for a parameterized name. In this sense, is `slugify` a misleading name for what I'm trying to do?